app/templates/index.html

```diff
@@ -4,45 +4,44 @@
 <head>
     <meta charset="UTF-8">
     <title>Wyze-Bridge</title>
-    <link rel="stylesheet" type="text/css" href="{{ url_for('static',filename='site.css') }}" />
-    {% if show_video %}
+    <link rel="stylesheet" type="text/css"
+        href="{{ 'static/site.css' if hass else url_for('static',filename='site.css') }}" />
     <script src="https://cdn.jsdelivr.net/npm/hls.js@1"></script>
-    {% endif %}
-    <script src="{{ url_for('static', filename='site.js') }}"></script>
+    <script src="{{ 'static/site.js' if hass else url_for('static', filename='site.js') }}"></script>
 </head>
 
 <body>
-    <h1>Wyze-Bridge</h1>
+    <h1>Wyze-Bridge v{{ version }}</h1>
     <h2>Detected {{cameras|length}} cameras:</h2>
     <label for="select_number_of_columns">Number of columns: </label>
     <input id="select_number_of_columns" type="number" min="1" max="10" value="{{ number_of_columns }}" />
 
     <section class="cameras">
         {% for camera in cameras.values() %}
         <div id="{{ camera.name_uri }}" draggable="true" class="camera">
-            <details open>
-                <summary>
-                    <div>{{camera.nickname}} -
-                        <a href="{{camera.hls_url}}">HLS</a>
-                        <a href="{{camera.rtmp_url}}">RTMP</a>
-                        <a href="{{camera.rtsp_url}}">RTSP</a>
-                        <span style="cursor: move; flex-grow: 2"></span>
-                        <details class="info">
-                            <summary>(info)</summary>
-                            <p>{{camera.__repr__()}}</p>
-                        </details>
-                        <span class="status {{'ONLINE' if camera.connected else 'OFFLINE'}}">{{'ONLINE' if
-                            camera.connected else 'OFFLINE'}}</span>
-                    </div>
-                </summary>
+            <div class="title">
```

Comment:
The extra div's are probably breaking the draggable logic in https://github.com/mrlt8/docker-wyze-bridge/blob/cb576d74e39d595b72fecc51d6887b1519b3b50a/app/static/site.js#L73 that looks for div